app/src/main/res/values/strings.xml

```diff
@@ -293,8 +293,8 @@
   <string name="overdue_empty_desc">You will be able to call patients who miss their scheduled visit for a follow-up blood pressure reading.</string>
   <string name="overdue_list_item_name_age">%s, %s</string>
   <plurals name="overdue_list_item_patient_bp_and_days_ago">
-  <item quantity="one">%1$s/%2$s, %3$s day ago</item>
-  <item quantity="other">%1$s/%2$s, %3$s days ago</item>
+    <item quantity="one">%1$s/%2$s, %3$s day ago</item>
+    <item quantity="other">%1$s/%2$s, %3$s days ago</item>
```

Comment:
Is this change committed by mistake?

Reply:
Those changes are made when I formatted the `strings.xml` file. So I made the commit with those changes present.